linux_os/guide/system/accounts/accounts-session/file_permissions_home_directories/oval/shared.xml

```diff
@@ -12,12 +12,21 @@
   <unix:password_object id="object_file_permissions_home_directories_objects" version="1">
     <unix:username datatype="string" operation="not equal">nobody</unix:username>
     <filter action="include">state_file_permissions_home_directories_interactive_uids</filter>
+  {{%- if product == 'rhel7' %}}
+    <filter action="exclude">state_file_permissions_home_files_permissions_nfsnobody</filter>
+  {{%- endif %}}
   </unix:password_object>
 
   <unix:password_state id="state_file_permissions_home_directories_interactive_uids" version="1">
     <unix:user_id datatype="int" operation="greater than or equal">{{{ uid_min }}}</unix:user_id>
   </unix:password_state>
 
+{{%- if product == 'rhel7' %}}
+  <unix:password_state id="state_file_permissions_home_files_permissions_nfsnobody" version="1">
+    <unix:username datatype="string" operation="equals">nfsnobody</unix:username>
```

Comment:
```suggestion
    <unix:username datatype="string" operation="equals">^(nobody|nfsnobody)$</unix:username>
```

Can't we do something like this and remove the `nobody` from line 13 and make this filter applicable to all products?

Reply:
nfsnobody shouldn't be expected to be a regular user even in other platforms, even though it should be technically possible

Reply:
We can, but I didn't want other products to have more users excluded than necessary.

Reply:
Hmm, maybe it is fine then.

Reply:
What is the conclusion? Should add `nobody` there or not?

Reply:
Let's do the following:

Line 13:
`<unix:username datatype="string" operation="pattern match">.*</unix:username>`

Then we keep the filter exclude for all platforms but we assign the list of users to a jinja variable ([example](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/rule.yml#L0-L1)) according to the product, for example:

rhel7 and ol7 - "nobody|nfsnobody"
other products - "nobody"

then

```suggestion
    <unix:username datatype="string" operation="pattern match">^({{{ user_list }}})$</unix:username>
```

Reply:
> rhel7 and ol7 - "nobody|nfsnobody"
> other products - "nobody"

I think we need achors in the regex, like:
`^(nobody|nfsnobody)$`

Reply:
Ahh, I see the anchors in the xml now, :)

Reply:
Where to define the variable with user list? I don't think `rule.yml` is a good place as it would be basically redundant.